### ISSUE TITLE
SYM-EVAL-001E-Q1S: preregistered statistical leakage-bound qualification

### ISSUE THREAD

**Issue by Tristan-Stoltz-ERC** [Luminous-Dynamics/symtropy#1291]
## Gate

Use this only for producer-visible channels where exact equality/deterministic canaries are not the whole theorem (for example noisy rendered features, timing distributions, learned branch probes, or cross-render-class equivalence).

Prefer exact byte/envelope equality whenever the profile supports it. Statistics must not weaken a theorem that can be checked exactly.

This is a benchmark-validity layer under SYM-EVAL-001E #1281, not a Symthaea performance test.

## Purpose

Prevent a common invalid inference:

```text
probe accuracy was not significantly above chance
    therefore
there is no leakage
```

Failure to reject above-chance classification is not evidence that leakage is sufficiently small.

For authoritative leak qualification, preregister a maximum tolerable **branch-inference advantage** and require an upper confidence bound to stay below it for the frozen probe family / population.

## Core claim form

For balanced blind A/B classification, define:

```text
chance_accuracy = 0.5
leak_advantage = accuracy - 0.5
```

Preregister a tolerance `epsilon`, e.g. conceptually:

```text
acceptable only if
one_sided_upper_confidence_bound(accuracy)
    <= 0.5 + epsilon
```

The exact epsilon, confidence level, interval method, sample size, and probe family must be frozen by CORE-004 before outcome inspection.

This establishes only a **probe-relative leakage bound**, never proof of zero leakage.

## Exact-vs-statistical hierarchy

Use the strongest available relation:

1. exact canonical-envelope equality;
2. exact content-hash equality within declared deterministic render class;
3. qualified numeric/perceptual tolerance relation;
4. preregistered statistical branch-inference bound.

Do not replace an exact failing equality check with a weaker classifier test.

## Grouped data splitting

Avoid train/test leakage across near-duplicate paired scenarios.

Split at the highest causal generation unit, such as:

```text
scenario family / world seed
```

not individual frames.

A/B branches from the same scenario family must not be divided across train and test in a way that lets the classifier memorize scenario identity and infer pairing artifacts.

Where multiple frames/prefixes come from one run, treat them as clustered/repeated observations for splitting and uncertainty.

## Frozen probe families

Before evaluation, freeze which probes exist and their tuning budget, for example:

```text
metadata-only deterministic rules
linear/logistic probe
small tree/forest probe
small MLP where justified
nearest-neighbor / similarity probe
ID-bit-pattern probe
timing-only probe
background-only probe
```

A post-hoc search across hundreds of classifiers followed by reporting only the reassuring ones is invalid.

If hyperparameter tuning is allowed, use nested or separate validation data and keep the final test set untouched.

## Canary power requirement

The leak detector is valid only if the frozen probe family detects preregistered intentional leaks of known strength.

Include canaries such as:

- one-pixel watermark;
- branch-coded timestamp offset;
- observation-ID bit leak;
- background cue;
- branch-coded path/metadata field;
- controlled timing delay.

For each canary profile preregister an expected minimum detection level or lower confidence bound.

If the clean profile looks like chance **and** the probe cannot reliably detect the canaries, report:

```text
ProbeInvalid / Underpowered
```

not `LeakChecksPass`.

## Sample-size / power planning

Choose sample count before seeing clean-profile outcomes.

The plan should bind at least:

- target maximum leak advantage `epsilon`;
- one-sided confidence level;
- desired power against a specified canary/leak advantage;
- number of scenario groups;
- number of prefixes/channels/probes;
- multiple-comparison correction policy.

Do not stop sampling when the confidence interval first becomes favorable unless that sequential rule was preregistered.

## Multiple comparisons

Q1/Q2/Q3 may test many:

- prefix lengths;
- modalities;
- metadata channels;
- probe families;
- render classes;
- scenario strata.

Freeze a simultaneous-error policy such as Holm/Bonferroni or another justified family-wise/FDR procedure.

A clean result at one prefix must not erase a leak at another.

Prefer reporting the full vector of channel/prefix bounds in addition to any family-level verdict.

## Prefix-time analysis

For each causal prefix `t`, estimate/bound branch-inference performance using only information legally available through `t`.

Expected shape:

```text
before first legal divergence:
    upper leak bound <= threshold

after legal divergence:
    branch information may legitimately increase
```

Do not pool post-divergence examples into a pre-divergence leakage test.

The first legal divergence itself must come from the sensor/causal-shielding theorem, not from the classifier result.

## Answerability and abstention

For probes or benchmark outputs with an explicit unknown/abstain option, keep these separate:

- branch classification accuracy;
- coverage / answer rate;
- confidence calibration;
- abstention behavior.

A probe that predicts only a tiny cherry-picked subset must not be compared to a full-coverage chance baseline without accounting for coverage.

## Permutation/randomization checks

Where appropriate, use scenario-group-preserving label permutations to validate the null distribution of the full probe pipeline.

Permutation must preserve the experimental unit and pairing structure; frame-level independent shuffling is invalid when frames share a scenario/run.

Permutation tests complement, but do not replace, the preregistered leakage-effect upper bound.

## Cross-hardware/render classes

Do not pool heterogeneous renderer/GPU classes unless comparability was qualified.

Report bounds by environment class first. A pooled statistic can hide a leak in one class and cancellation/noise in another.

Bind classes through CORE-006 environment identity.

## Suggested evidence output

For every `(profile, channel, prefix, probe_family, environment_class)` emit at least:

```text
n_scenario_groups
n_examples
class_balance
accuracy_or_metric
one_sided_upper_bound
leak_advantage_upper_bound
preregistered_epsilon
canary_detection_result
split_identity
probe/profile identity
multiple_test_family
status
```

Suggested statuses:

```text
LeakBoundPass
LeakBoundFail
LeakDetectedExact
ProbeInvalid
Underpowered
CanaryDetectionFail
ProfileNotComparable
InfrastructureNotExecuted
```

## Interpretation discipline

A pass means only:

> For the frozen probes, data population, channels, environment class, and statistical plan, the tested branch-inference advantage was bounded below the preregistered threshold.

It does **not** mean:

- zero mutual information;
- absence of unknown shortcuts;
- security against an arbitrarily powerful adversary;
- Symthaea performance;
- causal reasoning;
- real-world validity.

## Relationship to exact canaries

Current Q0/Q1 PR #1283 contains deterministic canaries and a known full-future leak. Those exact findings dominate any statistical result: a known deterministic leak cannot be "averaged away" by chance-level classifier performance elsewhere.

Q2 PR #1286 freezes canonical per-frame envelope equality for the synthetic shared prefix. That exact equality theorem should remain stronger than a learned-probe test for the same bytes.

Q1S becomes especially important once 001B introduces noisy/rendered modalities where exact equality may require renderer-class qualification or a separately frozen equivalence relation.

## Relationships

- anti-shortcut umbrella: #1281
- Q0/Q1 detector: PR #1283
- Q2 producer envelope: PR #1286
- Q3 metamorphics: #1290
- Q4 scorer manipulation: #1288
- preregistration: #1237 / CORE-004
- environment identity: #1256 / CORE-006
- rendered sensor adapter: #1215 / SYM-EVAL-001B
